src/reusable/card-image.js

```diff
@@ -3,11 +3,11 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { SPACING } from '../reusable';
 
-export default function CardImage ({ image }) {
+export default function CardImage ({ image, alt }) {
   return (
     <GatsbyImage
       image={image}
-      alt={image.alt || ''}
+      alt={alt || ''}
```

Comment:
```suggestion
export default function CardImage ({ image, alt = '' }) {
  return (
    <GatsbyImage
      image={image}
      alt={alt}
```

Not a blocker, the `alt` parameter could be defined with the default value of an empty string.

Reply:
Added and changed. Committing the suggestion from the github UI isn't erroring out so I changed it locally and pushed.